MAP-LIB_ProjectRules.md

```diff
@@ -276,9 +276,9 @@ Members:
   | `Mechanics.MultiBody`        | Martin Otter, Jakub Tobolar                                     |
   | `Mechanics.Rotational`       | Anton Haumer, Christian Kral, Martin Otter, Jakub Tobolar       |
   | `Mechanics.Translational`    | Anton Haumer, Christian Kral, Martin Otter, Jakub Tobolar       |
-  | `Fluid`                      | Francesco Casella, Rüdiger Franke, Hubertus Tummescheit         |
-  | `Fluid.Dissipation`          | Francesco Casella, Stefan Wischhusen                            |
-  | `Media`                      | Francesco Casella, Rüdiger Franke, Hubertus Tummescheit         |
+  | `Fluid`                      | Francesco Casella, Arunkumar Narashiman                         |
+  | `Fluid.Dissipation`          | Francesco Casella                                               |
```

Comment:
This violates 5.1.2.1

> For every library or sublibrary covered by MAP-LIB , two or more Library Officers are assigned ...

Reply:
If the special officer for Fluid.Dissipation no longer wants to contribute I would propose to not list Fluid.Dissipation separately instead of breaking 5.1.2.1.

And similarly for Magnetic skip the sub-libraries and only list Tony and Christian Kral for Magnetic; see mail January 9th/11th titled "AW: Library Officers for Modelica.Magnetic.FluxTubes".